src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -1269,6 +1270,7 @@ private String getWebappImageResource(String imageName) {
     })
     @Tag(name = "saveAuxiliaryFileWithVersion", 
          description = "Save Auxiliary File With Version")
+    @SecurityRequirement(name = "api_token")
```

Comment:
Why is it "api_token" and not "API token" or "API_Token" or whatever else? Is it arbitrary?

Reply:
Seems that we also have to define the "security schema" and we name that security schema, we also set a type. The security schema could be named something else but in this case api_key is the name